w1

WebDevLyna opened the pull request:
Pull request for week 1

Commit:
w1

Commit:
Adding bootstrap buttons

WebDevLyna committed:
Updating index file

Adding meta names, adding description, font awesome cdn and icons, Resume button.

WebDevLyna committed:
Styling icons and button

Add styles to font awesome icons and bootstrap button.

WebDevLyna committed:
Adding js scripts, revamp some divs, style clean up

Commit:
Merge branch 'master' of https://github.com/calvintennant/gbc_comp-9635_03

Commit:
Merge branch 'master' of https://github.com/calvintennant/gbc_comp-9635_03

Commit:
Adding package.json, index.js, index.html

deleted lastname-firstname folder

Commit:
Adding script.js and style.css files

Commit:
Adding Google Maps API

No need for jQuery, just google maps and google.maps.event.addDomListener(window, 'load', function()...

Commit:
Adding the final commit in class. Will work on the childhood location for next week.

calvinte reviewed (changes requested):
It looks like you've made some changes that I can't approve. 

The files inside of `week_1/lastname-firstname` should remain unchanged (they've been moved).

Please review. If you have any difficulty with this please see me in class. Thanks!

WebDevLyna commented:
Hi Calvin,

Yeah, I realize I had deleted that file thinking I didn't need it, but I
didn't realize it would affect your end. Sorry! In order to not mess
anything up, I will need some assistance in class on Thursday :).


Thank you,

Lyna

On Sat, Feb 3, 2018 at 1:13 PM, Calvin Tennant <notifications@github.com>
wrote:

> *@calvintennant* requested changes on this pull request.
>
> It looks like you've made some changes that I can't approve.
>
> The files inside of week_1/lastname-firstname should remain unchanged
> (they've been moved).
>
> Please review. If you have any difficulty with this please see me in
> class. Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/calvintennant/gbc_comp-9635_03/pull/4#pullrequestreview-93823211>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGM4ssoNSXl473GoZlOURIVc7gE3WU0aks5tRKHEgaJpZM4Rtt-1>
> .
>

WebDevLyna committed:
Google Maps picks up value input

WebDevLyna committed:
Adding some styles to map

WebDevLyna committed:
adding lastname-firstname back in

Commit:
Merge branch 'master' of https://github.com/calvintennant/gbc_comp-9635_03

Commit:
Week 3 - Initial commit

Commit:
Adding first travelled object

San Francisco. Also adding map and loop as well as marker.

Commit:
Adding the rest of the destinations as well as images

WebDevLyna committed:
Adding styles to infowindow as well as some description edits

Commit:
Merge branch 'master' of https://github.com/calvintennant/gbc_comp-9635_03

Commit:
Adding FourSquare API AJAX

Adding Google Static Maps as well as Query Parameters for AJAX to make a call for images and marker.
